pipeline/filters/lua.md

```diff
@@ -180,7 +176,22 @@ pipeline:
 {% endtab %}
 {% endtabs %}
 
-#### Environment variable processing
+### Number Type
+
++Lua treats number as double. It means an integer field (e.g. IDs, log levels) will be converted double. To avoid type conversion, The `type_int_key` property is available.
+
+### Protected Mode
+
+Fluent Bit supports protected mode to prevent crash when executes invalid Lua script. See also [Error Handling in Application Code](https://www.lua.org/pil/24.3.1.html).
```

Comment:
```suggestion
Fluent Bit supports protected mode to prevent crashes if it executes an invalid Lua script.
See also [Error Handling in Application Code](https://www.lua.org/pil/24.3.1.html).
```